doc/nrf/gs_installing.rst

```diff
@@ -51,7 +51,19 @@ The installation process is different depending on your operating system.
          :docset: zephyr
          :dedent: 6
          :start-after: .. _install_dependencies_windows:
-         :end-before: #. Close the window and open a new
+         :end-before: #. Use ``choco`` to install the required
+
+
+      4. Use ``choco`` to install the required dependencies:
+         
+         .. parsed-literal::
+            :class: highlight
+
+            choco install cmake --installargs 'ADD_CMAKE_TO_PATH=System'
+            choco install ninja --version= *ninjaversion* gperf --version= *gperfversion* python --version= *pythonversion* git --version= *gitversion*
+         
+         where *ninjaversion*, *gperfversion*, *pythonversion* and *gitversion* correspond to the versions in the :ref:`Required tools table <req_tools_table>`.
```

Comment:
I would place this in line 57:
`      4. Use ``choco`` to install the required dependencies, with *ninjaversion*, *gperfversion*, *pythonversion* and *gitversion* corresponding to the versions in the :ref:`Required tools table <req_tools_table>`:`

Reply:
I kept the original content because it's rendering from zephyr and any changes there won't be reflected if I add it as a separate step.
There is now a note, as its important to install the tools with versions specified in the 'Required Tools' section, or else there may be issues later.